app/clients/email/stub_email.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from flask import current_app
+from requests import request
+from time import monotonic
+
+from app.clients.email import (EmailClientException, EmailClient)
+
+
+class AwsSesStubClientException(EmailClientException):
+    pass
+
+
+class AwsSesStubClient(EmailClient):
+    def init_app(self, region, statsd_client, stub_url):
+        self.name = 'ses'
+        self.statsd_client = statsd_client
+        self.url = stub_url
+
+    def get_name(self):
+        return self.name
+
+    def send_email(self,
+                   source,
+                   to_addresses,
+                   subject,
+                   body,
+                   html_body='',
+                   reply_to_address=None):
+        try:
+            start_time = monotonic()
+            response = request(
+                "POST",
+                self.url,
+                data={"id": "dummy-data"},
```

Comment:
we had a brief discussion if we want to send realistic data to the stub, but we have not reached a conclusion, right? What do you gather?